agimus_controller_ros/test/test_order_5_polynomial.py

```diff
@@ -16,13 +16,15 @@ def tearDownClass(cls):
         return super().tearDownClass()
 
     def test_quintic_trajectory(self):
-        obj = SimpleTrajectoryPublisher()
+        amp = 0.5
+        scale_duration = 0.7
+        obj = QuinticTrajectory(scale_duration=scale_duration, amp=amp)
         times = np.linspace(0, obj.scale_duration, 200)
-        positions = [obj.quintic_trajectory(t) for t in times]
+        positions = [obj.get_value_at_t(t)[0] for t in times]
```

Comment:
You could also check the derivatives. E.g. using `scipy.optimize.approx_fprime` or ``scipy.optimize.check_grad` or a home-made finite difference algo.